spec/plaindatetime.html

```diff
@@ -840,31 +840,60 @@ <h1>Properties of Temporal.PlainDateTime Instances</h1>
   <emu-clause id="sec-temporal-plaindatetime-abstract-ops">
     <h1>Abstract operations</h1>
 
-    <emu-clause id="sec-temporal-getepochfromisoparts" aoid="GetEpochFromISOParts">
-      <h1>GetEpochFromISOParts ( _year_, _month_, _day_, _hour_, _minute_, _second_, _millisecond_, _microsecond_, _nanosecond_ )</h1>
+    <emu-clause id="sec-temporal-getepochfromisoparts" type="abstract operation">
+      <h1>
+        GetEpochFromISOParts (
+          _year_: an integer,
+          _month_: an integer between 1 and 12 inclusive,
+          _day_: an integer between 1 and 31 inclusive,
+          _hour_: an integer between 0 and 23 inclusive,
+          _minute_: an integer between 0 and 59 inclusive,
+          _second_: an integer between 0 and 59 inclusive,
+          _millisecond_: an integer between 0 and 999 inclusive,
+          _microsecond_: an integer between 0 and 999 inclusive,
+          _nanosecond_: an integer between 0 and 999 inclusive,
+        ): a BigInt
```

Comment:
This was treated as `integer` in some places and `BigInt` in others. I picked `BigInt` but could go either way.